### ISSUE TITLE
Cryptonight V8

### ISSUE THREAD

**Issue by Auserone** [xmrMiner/xmrMiner-Win64#17]
Cryptonight V8 would be very nice for xmrminer.

**Comment by mmahdim** [comment]
Yeah, this is the only miner that supports cuda 8 for my old GPU. But seems like ccminer team are not planing to update it.
